src/vic3_world/pacts/pact_importer_test.cpp

```diff
@@ -0,0 +1,38 @@
+#include "src/vic3_world/pacts/pact_importer.h"
+
+#include <sstream>
+
+#include "external/commonItems/external/googletest/googlemock/include/gmock/gmock-matchers.h"
+#include "external/commonItems/external/googletest/googletest/include/gtest/gtest.h"
+
+namespace vic3
+{
+
+TEST(Vic3WorldPactImporter, PactImporter)
+{
+   PactImporter importer;
+   std::stringstream input;
+   input << R""(
+{
+	first=1053
+	second=268
+	action=alliance	start_date=1930.2.27
+	forced_duration=120
+}
+)"";
+   Pact pact = importer.ImportPact(0, input);
+   EXPECT_EQ(pact, Pact(1053, 268, "alliance", date("1930.2.27"), 120));
+   EXPECT_FALSE(pact.isSubjectRelationship());
```

Comment:
Since this functionality is in pact.h, this should be moved into a new test in a file tied to pact.h and not the importer.